### ISSUE TITLE
MAE metric seems get incorrect result

### ISSUE THREAD

**Issue by Litchilitchy** [intel-analytics/analytics-zoo#1172]
In my example, with the same loss and accuracy, and all the same parameters, with metric MAE, the compile code is

    model.compile(optimizer='rmsprop', loss='mse', metrics=['mae'])

but the MAE result of this model is 10 times larger than the same model in Keras 

Currently I am not very sure whether it is affected by other configurations, please contact me if you can not re-implement this wrong result

**Comment by Litchilitchy** [comment]
fixed [here](https://github.com/intel-analytics/analytics-zoo/pull/1232)
